composer.json

```diff
@@ -13,13 +13,9 @@
     },
     "scripts": {
         "install-bats": "if [ ! -f bin/bats ] ; then git clone https://github.com/sstephenson/bats.git; mkdir -p $HOME/bin; bats/install.sh $HOME; fi",
-        "install-phpcs": "mkdir -p phpcs && cd phpcs && COMPOSER_BIN_DIR=../vendor/bin composer require squizlabs/php_codesniffer:^2.7",
-        "bats": "TERMINUS_PLUGINS_DIR=.. PATH=$HOME/bin:$PATH bats -p -t tests",
+        "install-phpcs": "mkdir -p phpcs && COMPOSER_BIN_DIR=../vendor/bin composer require -d ./phpcs squizlabs/php_codesniffer",
+        "bats": "bats -p -t tests",
```

Comment:
How does `bats` execute without the full path?

Reply:
The docker image has the binary in the PATH